acceptance/acceptance_test.go

```diff
@@ -267,24 +254,56 @@ func runTest(t *testing.T, dir, coverDir string, repls testdiff.ReplacementsCont
 		server.RecordRequests = config.RecordRequests
 		server.IncludeRequestHeaders = config.IncludeRequestHeaders
 
-		// If no custom server stubs are defined, add the default handlers.
-		if len(config.Server) == 0 {
-			AddHandlers(server)
-		}
-
 		for _, stub := range config.Server {
 			require.NotEmpty(t, stub.Pattern)
-			server.Handle(stub.Pattern, func(fakeWorkspace *testserver.FakeWorkspace, req *http.Request) (any, int) {
+			items := strings.Split(stub.Pattern, " ")
+			require.Len(t, items, 2)
+			server.Handle(items[0], items[1], func(fakeWorkspace *testserver.FakeWorkspace, req *http.Request) (any, int) {
 				statusCode := http.StatusOK
 				if stub.Response.StatusCode != 0 {
 					statusCode = stub.Response.StatusCode
 				}
 				return stub.Response.Body, statusCode
 			})
 		}
-		cmd.Env = append(cmd.Env, "DATABRICKS_HOST="+server.URL)
+
+		// The earliest handlers take precedence, add default handlers last
+		AddHandlers(server)
+
+		databricksHost = server.URL
+	}
+
+	databricksToken := os.Getenv("DATABRICKS_TOKEN")
+
+	// Each local test should use a new token that will result into a new fake workspace,
+	// so that test don't interfere with each other.
+	if cloudEnv == "" {
+		tokenSuffix := strings.ReplaceAll(uuid.NewString(), "-", "")
+		databricksToken = "dbapi" + tokenSuffix
 	}
 
+	workspaceClient, err := databricks.NewWorkspaceClient(&databricks.Config{
+		Host:  databricksHost,
+		Token: databricksToken,
+	})
+	require.NoError(t, err)
+
+	cmd.Env = append(cmd.Env, "DATABRICKS_HOST="+databricksHost)
+	cmd.Env = append(cmd.Env, "DATABRICKS_TOKEN="+databricksToken)
+
+	ctx := context.Background()
+	if cloudEnv == "" {
+		testdiff.PrepareReplacementsUser(t, &repls, testUser)
+	} else {
+		user, err := workspaceClient.CurrentUser.Me(ctx)
```

Comment:
Why are we making this API call?

Reply:
Do we no longer need to replace users for the integration tests?

Reply:
good point - we do need it, it was a bug. Fixed in 745117faad4669376a4ec2555914da04dc4e2637